docker/docker-caching/multi-module-caching/pom.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung</groupId>
+    <artifactId>multi-module-caching</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>maven-caching</name>
```

Comment:
Note that the name node is not required, but if we include it, it must be named the same as the artifactId (in accordance with the guidelines).